### DISCUSSION TITLE
Welcome to hydra Discussions!

### DISCUSSION THREAD

**Discussion by coder46** [georgianpartners/hydra#71] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jichaoz** [comment]
What is the mission of this project? There are a lot we can do in MLOps. What will be in scope and what will be out of scope?
